src/mappings/ro-to-sio.sssom.tsv

```diff
@@ -0,0 +1,37 @@
+#curie_map:
+#  SIO: http://semanticscience.org/resource/SIO_
+#  RO: http://purl.obolibrary.org/obo/RO_
+#  BFO: http://purl.obolibrary.org/obo/BFO_
+#  skos: http://www.w3.org/2004/02/skos/core#
+#license: https://creativecommons.org/publicdomain/zero/1.0/
+#mapping_provider: http://obofoundry.org/ontology/ro
+#mapping_set_group: ro-sio-mappings

```

Comment:
group is not yet official, do you mean perhaps mapping_set_id?

Reply:
yeh I'll just take this out for now